executor/src/witgen/jit/effect.rs

```diff
@@ -23,6 +23,8 @@ pub enum Effect<T: FieldElement, V> {
     Assertion(Assertion<T, V>),
     /// A call to a different machine, with identity ID, known inputs and argument variables.
     MachineCall(u64, BitVec, Vec<V>),
+    /// Compute one variable by executing a prover function (given by index) on the value of other variables.
```

Comment:
maybe not for this PR, but this enum might be a good candidate for using named fields